src/System.Numerics.Matrices/src/System/Numerics/Matrix5x5.cs

```diff
@@ -0,0 +1,543 @@
+        /// <summary>
+        /// Gets a new Matrix1x5 containing the values of column 4
+        /// </summary>
+        public Matrix1x5 Column4 { get { return new Matrix1x5(M41, M42, M43, M44, M45); } }
+        /// <summary>
+        /// Gets a new Matrix1x5 containing the values of column 5
+        /// </summary>
+        public Matrix1x5 Column5 { get { return new Matrix1x5(M51, M52, M53, M54, M55); } }
+        /// <summary>
+        /// Gets a new Matrix5x1 containing the values of column 1
+        /// </summary>
+        public Matrix5x1 Row1 { get { return new Matrix5x1(M11, M21, M31, M41, M51); } }
+        /// <summary>
+        /// Gets a new Matrix5x1 containing the values of column 2
+        /// </summary>
+        public Matrix5x1 Row2 { get { return new Matrix5x1(M12, M22, M32, M42, M52); } }
+        /// <summary>
+        /// Gets a new Matrix5x1 containing the values of column 3
+        /// </summary>
+        public Matrix5x1 Row3 { get { return new Matrix5x1(M13, M23, M33, M43, M53); } }
+        /// <summary>
+        /// Gets a new Matrix5x1 containing the values of column 4
+        /// </summary>
+        public Matrix5x1 Row4 { get { return new Matrix5x1(M14, M24, M34, M44, M54); } }
+        /// <summary>
+        /// Gets a new Matrix5x1 containing the values of column 5
+        /// </summary>
+        public Matrix5x1 Row5 { get { return new Matrix5x1(M15, M25, M35, M45, M55); } }
+
+        public override bool Equals(object obj)
+        {
+            if (obj is Matrix5x5)
+                return this == (Matrix5x5)obj;
+
+            return false;
+        }
+
+        public bool Equals(Matrix5x5 other)
+        {
+            return this == other;
+        }
+
+        public unsafe override int GetHashCode()
+        {
+            fixed (Matrix5x5* p = &this)
+            {
+                int* x = (int*)p;
+                unchecked
+                {
+                    return 0xFFE1
+                         + 7 * ((((x[00] ^ x[01]) << 0) + ((x[02] ^ x[03]) << 1) + ((x[04] ^ x[05]) << 2) + ((x[06] ^ x[07]) << 3) + ((x[08] ^ x[09]) << 4)) << 0)
+                         + 7 * ((((x[05] ^ x[06]) << 0) + ((x[07] ^ x[08]) << 1) + ((x[09] ^ x[10]) << 2) + ((x[11] ^ x[12]) << 3) + ((x[13] ^ x[14]) << 4)) << 1)
+                         + 7 * ((((x[10] ^ x[11]) << 0) + ((x[12] ^ x[13]) << 1) + ((x[14] ^ x[15]) << 2) + ((x[16] ^ x[17]) << 3) + ((x[18] ^ x[19]) << 4)) << 2)
+                         + 7 * ((((x[15] ^ x[16]) << 0) + ((x[17] ^ x[18]) << 1) + ((x[19] ^ x[20]) << 2) + ((x[21] ^ x[22]) << 3) + ((x[23] ^ x[24]) << 4)) << 3)
+                         + 7 * ((((x[20] ^ x[21]) << 0) + ((x[22] ^ x[23]) << 1) + ((x[24] ^ x[25]) << 2) + ((x[26] ^ x[27]) << 3) + ((x[28] ^ x[29]) << 4)) << 4);
+                }
+            }
+        }
+
+        public override string ToString()
+        {
+            return "Matrix5x5: "
```

Comment:
I originally did this with `String.Format(...)` and having the concat strings was "free" as they should have been concatenated at compile time. With interpolated strings, does it mean a bunch of strings will get alloc'd then concatenated on each `ToString` invocation?